integration/fields.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package integration
+
+func isWhitespace(b byte) bool {
+	return b == '\t' || b == '\n' || b == '\v' ||
+		b == '\f' || b == '\r' || b == ' '
+}
+
+// fields splits the string s around each instance of one or more consecutive white space
+// characters, returning a slice of substrings of s or an
+// empty slice if s contains only white space.
+// fields doesn't split substring of s if substring is inside of quotes.
+func fields(s string) []string {
```

Comment:
note how this is different from strings.Fields and why

Reply:
added comment